iocore/net/SSLUtils.cc

```diff
@@ -1866,10 +1934,17 @@ ssl_store_ssl_context(const SSLConfigParams *params, SSLCertLookup *lookup, cons
     keyblock = ssl_context_enable_tickets(ctx, NULL);
   }
 
+  // Generate HPKP header if hpkp is enabled.
+  if (sslMultCertSettings.hpkp_enabled >= 0 ? sslMultCertSettings.hpkp_enabled : params->hpkp_enabled) {
```

Comment:
It is possible:) `sslMultCertSettings.hpkp_enabled` is initialized by `-1`, so settings in ssl_multicert.config (`0` or `1`) is preceded if they're written. This is expected behavior, right?